docs/pages/User/Install/pre-install.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Pre-Install
+weight: 2
+---
+
+### Introduction
+
+Liqo can be installed either in private or local clusters. Its configuration depends on the type of connectivity between the two clusters. Before installing Liqo, you have to consider how your clusters can connect and can peer together.
+
+### Peering Requirements
+
+Liqo requires the following services to be reciprocally accessible on both clusters to be able to start the cluster peering:
+
+* **Authentication server**: Liqo service, used to authenticate incoming peering requests coming from other clusters (i.e., `liqo-auth`). You should modify the values in section ``auth`` in the [Liqo chart values](/user/install/chart_values) to configure how the authentication server is exposed.
+* **Kubernetes API server**: Kubernetes APIServer, where the (remote) Liqo instance will create some new resources when the peering process starts. APIServer can be configured in the ``apiServer`` section of the [Liqo chart values](/user/install/chart_values). By default, Liqo will use an endpoint composed of the IP of the first master and the 6443 port. In managed clusters, you have to configure those values to have Liqo working correctly.
+* **Network gateway**: Liqo service responsible for setting up the inter-cluster connectivity between clusters (i.e., `liqo-gateway`). The Liqo Gateway is configured in the ``gateway`` section of the [Liqo chart values](/user/install/chart_values).
+
+Depending on the physical setup of your cluster, you need to properly configure some parameters required by Liqo during the install process to enable remote clusters to contact the above services. Below we present some common scenarios that Liqo can handle. Once you identify yours, you can go have to the *table* of each section to find the right values you should specify when installing Liqo using the chart.
+
+The exposition parameters can be configured at installation time using the [Liqo Helm Chart](/user/install/chart_values) and updated after the installation by issuing an ``helm update`` after changing them in your values.yml. If you need more information about Helm and how charts can be configured, you can have a look at the [Helm official documentation](https://helm.sh/docs/). Pay attention that changing exposition parameters may affect and break active peerings. We suggest disabling all peerings before changing the way a cluster is exposed.
+
+### Cloud to cloud
+
+![](/images/scenarios/cloud-to-cloud.svg)
+
+Two managed clusters peered together through the Internet. It is possible to have a multi-cloud setup (AKS to AKS, GKE to GKE, and AKS to GKE). In this scenario, the services to exposes should be exposed using Public IPs/URLs.
+
+|  | Cluster A (Cloud) | Cluster B (Cloud) |
+| --------- | -------- |  ---------       | |
+| **Auth Server** |  LoadBalancer/ingress | LoadBalancer/Ingress |
+| **API server** | Provided | Provided |
+| **Network gateway** | LoadBalancer/Node Port | LoadBalancer/Node Port |
```

Comment:
Two comments here. 
1. The table seems broken (at least in the GitHub preview)
2. What does Provided mean?